File: 3-semestre/.NET/Aula 07/Exercicio03/ContaCorrente.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Exercicio03
{
    internal class ContaCorrente : ContaBancaria
    {
        public double deposito;
        public double saque;
        public override void Depositar()
        {
            saldo = saldo + deposito;
            Console.WriteLine("Depositado!");
        }
        public override void Sacar()
        {
            if (saque <= saldo)
            {
                saldo = saldo - saque;
                Console.WriteLine("Sacado!");
            } 
            else
            {
                Console.WriteLine("Saldo insuficiente.");
            }
        }

    }
}

File: 3-semestre/programação orientada a objetos/abstrata_interface/exercicios/ex3/ContaCorrente.java
package ex3;

public class ContaCorrente extends Conta {

	@Override
	void depositar(double deposito) {
		saldo += deposito;
	}

	@Override
	void sacar(double saque) {
		if(saque < limite + saldo) {
			saldo -= saque;	
		}
	}

}